codemods/index.js

```diff
@@ -0,0 +1,8 @@
+const coolButton = require('./cool-button');
```

Comment:
It's good that we committed these now, but we will probably want to remove them later on. Maybe we should have an example code mod?